code_challenges.rb

```diff
@@ -20,10 +32,40 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# Expected input: string
+# use puts to return a value
+# add variable 
+# use method called .delete on end of variable
+# delete any vowels within the string
+
+
+puts album1.delete "aeiou"
+puts album2.delete "aeiou"
+puts album3.delete "aeiou"
```

Comment:
While this totally works, it's not a custom method we can call on say in a larger project. What elements of writing a method would you have to account for in a custom method that does the same work?